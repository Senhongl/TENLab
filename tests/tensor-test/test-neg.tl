a = -10;
print(a);
b = -[-1.2, 2.3, -0.5];
print(b);