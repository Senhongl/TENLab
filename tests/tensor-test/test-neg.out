-10
[ CPUIntType{} ]
 1.2000
-2.3000
 0.5000
[ CPUDoubleType{3} ]
